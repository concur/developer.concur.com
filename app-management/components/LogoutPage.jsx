import React from 'react';
import { Link } from 'react-router';

const Logout = () => (
  <div>
    <h2>You are logged out</h2>
    <Link to="/login">Login</Link>
  </div>
);

export default Logout;
