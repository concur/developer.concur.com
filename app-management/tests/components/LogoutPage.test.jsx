import React from 'react';
import { shallow } from 'enzyme';
import { Link } from 'react-router';
import { shallowToJson } from 'enzyme-to-json';
import LogoutPage from '../../components/LogoutPage';

describe('<LogoutPage />', () => {
  it('renders with logout text and link to the login page', () => {
    const page = shallow(<LogoutPage />);

    expect(shallowToJson(page)).toMatchSnapshot();
  });
});
