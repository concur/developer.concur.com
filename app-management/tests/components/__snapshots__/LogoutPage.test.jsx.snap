exports[`<LogoutPage /> renders with logout text and link to the login page 1`] = `
<div>
  <h2>
    You are logged out
  </h2>
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/login">
    Login
  </Link>
</div>
`;
